jobs/JEVS_GLOBAL_ENS_PREP

```diff
@@ -38,16 +38,17 @@ export DCOMIN=${DCOMIN:-${DCOMROOT}}
 export COMINccpa=${COMINccpa:-$(compath.py $envir/com/ccpa/${ccpa_ver})}
```

Comment:
@BinbinZhou-NOAA I think that COMINgfsanl needs to be just COMINgfs.  Since the rules are COMINxxx must be where xxx=a model, the model is gfs and not gfsanl.  Can we make this change here and elsewhere it uses COMINgfsanl?